frontend/package.json

```diff
@@ -17,7 +17,7 @@
     "lint": "yarn eslint .",
     "test": "jest",
     "debug-test": "node --inspect-brk node_modules/.bin/jest --runInBand",
-    "webdriver-update": "webdriver-manager update --gecko=false",
+    "webdriver-update": "webdriver-manager update --gecko=true",
```

Comment:
Just remove the `--gecko` flag altogether.

Reply:
Updated.